distribution/feature-packs/server-feature-pack/src/main/resources/licenses/rh-sso/Select2,Apache Software License 2.0.txt

```diff
@@ -0,0 +1 @@
+Select2,GNU General Public License v2.0 only.txt
```

Comment:
That's true for the modern versions, but the version in use in 7.1.0 is licensed as ASL v2 or GPL v2. https://github.com/select2/select2/blob/3.4.1/LICENSE